pkg/controller/operators/catalog/operator.go

```diff
@@ -1329,6 +1344,27 @@ func EnsureCRDVersions(oldCRD *apiextensionsv1.CustomResourceDefinition, newCRD
 	return nil
 }
 
+// Ensure all existing served versions are present in new CRD
+func EnsureV1Beta1CRDVersions(oldCRD *apiextensionsv1beta1.CustomResourceDefinition, newCRD *apiextensionsv1beta1.CustomResourceDefinition) error {
```

Comment:
why is this an issue? Versions can go away, like old alpha versions.

Reply:
This a good point and we have addressed it by removing this function entirely.